pkg/storage/tsdb/block/meta.go

```diff
@@ -3,7 +3,7 @@
 // Provenance-includes-license: Apache-2.0
 // Provenance-includes-copyright: The Thanos Authors.
 
-package metadata
+package block
 
 // metadata package implements writing and reading wrapped meta.json where Thanos puts its metadata.
```

Comment:
This comment mentions the name of the package. It can be changed to mention the name of the file instead